src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/model/entity/session/category/PrisonerCategory.kt

```diff
@@ -0,0 +1,25 @@
+package uk.gov.justice.digital.hmpps.visitscheduler.model.entity.session.category
+
+/*
+TODO - this needs to be reconsidered based on tests and further analysis and might need to be replaced with a String.
+ */
+enum class PrisonerCategory(
```

Comment:
We have generally call these PrisonerCategoryType but your call

Reply:
will update to PrisonerCategoryType